### ISSUE TITLE
npm run dev error

### ISSUE THREAD

**Issue by abdokouta** [braginteractive/VueWPLeads#1]
hello, thanks for that awesome repo., but unfortunately i can't run it when i'm try to run "npm run dev" after installing all the dependencies i got mapGetter error so i have updated all dependencies in package.json and install the updated dependencies once again and i got css loader error, i have tried several time to solve this but i couldn't so what is wrong with the mappGetter or the css. could u please help?

**Comment by braginteractive** [comment]
Sorry, can't really provide any insight without the errors you are getting. I would also recommend Googling the errors to see how others are fixing the issues.

**Comment by abdokouta** [comment]
> Sorry, can't really provide any insight without the errors you are getting. I would also recommend Googling the errors to see how others are fixing the issues.

i already have googling but i didn't get any solution for it, did u tried the code and it works?

**Comment by braginteractive** [comment]
Yes, do you have everything else set up to run with this codebase? https://braginteractive.com/wordpress-lead-dashboard-with-vue-js-and-contact-7/

**Comment by thebarton** [comment]
The readme.md doesn't really make it clear that you need to do more than just type in npm run dev for it to work. The ...mapGetter issue is with babel missing a couple files. 
It needs at least 2 files (so far, other than the additional plugins):

- .babelrc
- .postcssrc.js

.babelrc
`{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
      }
    }],
    "stage-2"
  ],
  "plugins": ["transform-vue-jsx", "transform-runtime"],
  "env": {
    "test": {
      "presets": ["env", "stage-2"],
      "plugins": ["transform-vue-jsx", "transform-es2015-modules-commonjs", "dynamic-import-node"]
    }
  }
}`

.postcssrc.js 
`// https://github.com/michael-ciniawsky/postcss-load-config

module.exports = {
  "plugins": {
    "postcss-import": {},
    "postcss-url": {},
    // to edit target browsers: use "browserslist" field in package.json
    "autoprefixer": {}
  }
}
`

**Comment by braginteractive** [comment]
@thebarton have you tried Googleing your NPM errors? This isn't a theme error. This is a tooling error. The npm packages probably just need to be updated. 

Please keep us updated.

**Comment by thebarton** [comment]
I was answering the original ticket question. Those 2 files fix the issue that he/she was having.

**Comment by braginteractive** [comment]
Oh, I see. Thanks for helping.
